osu.Game/Overlays/KeyBinding/VariantBindingsSubsection.cs

```diff
@@ -7,8 +7,7 @@ namespace osu.Game.Overlays.KeyBinding
 {
     public class VariantBindingsSubsection : KeyBindingsSubsection
     {
-        protected override string Header => variantName;
-        private readonly string variantName;
+        protected override string Header { get; }
```

Comment:
Same thing here, would rather have `VariantName` back and `Header` delegating to it myself.

Reply:
I prefer this personally. Bit of a much of a muchness.